extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/lib/config/Config.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.digitalstrom.internal.lib.config;
+
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.sensorJobExecuter.sensorJob.SensorJob;
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.structure.devices.Device;
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.structure.devices.deviceParameters.OutputModeEnum;
+
+/**
+ * The {@link Config} contains all configurations for the digitalSTROMLibary.<br>
+ * For that it gives default constants and variables that can be set during the run-time.
+ *
+ * @author Michael Ochel - Initial contribution
+ * @author Matthias Siegele - Initial contribution
+ */
+public class Config {
```

Comment:
Note, as mentioned above:
This should be a proper Java bean, best with getters and setters and private internal instance fields.